base_comment_template/models/base_comment_template.py

```diff
@@ -15,69 +16,46 @@ class CommentTemplate(models.AbstractModel):
         "base.comment.template to put header and footer "
         "in reports based on created comment templates"
     )
+    """
+    It's for allowing to have another field that determines the source partner for
+    the comment templates.
+    Imagine other models where the partner field is called customer_id for example.
+    """
+    _comment_template_partner_field_name = "partner_id"
```

Comment:
Thanks for the comment @victoralmau 
However, I think this would need a comment, not really a docstring for the class:
```suggestion
    # This field allows to set any given field that determines the source partner for
    # the comment templates downstream.
    # E.g.: other models where the partner field is called customer_id.
    _comment_template_partner_field_name = "partner_id"
```